addons/website/views/snippets/s_adventure.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_adventure" name="Adventure">
+    <section class="s_adventure pt200 pb200 oe_img_bg" style="background-image: url('/web/image/website.s_adventure_default_image'); background-position: 50% 0%;">
+        <div class="o_we_bg_filter bg-white-50"/>
+        <div class="container">
+            <h1 style="text-align: center;">
```

Comment:
As discussed, headings are not editable   ;)


https://github.com/user-attachments/assets/dda43e4a-eb88-43b0-bfc7-47933865376a